package.json

```diff
@@ -15,7 +15,9 @@
     "jquery-ujs": "^1.2.2",
     "jquery.turbolinks": "^2.1.0",
     "js-cookie": "^2.2.0",
+    "js-polynomial-regression": "^0.9.0",
     "moment": "^2.20.1",
+    "moment-duration-format": "^2.2.2",
```

Comment:
I'm so glad someone made this finally instead of having to hack around with moment objects.

Reply:
Yeah seriously, that was insane. This is so much better.